include/libfdt/fdt.h

```diff
@@ -0,0 +1,111 @@
+#ifndef _FDT_H
```

Comment:
It's better (for legal reasons) if this 3rd party IP is imported into the codebase by us (ARM). We already have approval for importing libfdt so I will arrange this. Then you will be able to remove this commit from the PR.